examples/serialization/avro/avro-cli.py

```diff
@@ -19,61 +19,53 @@
 
 from six.moves import input
 
-from confluent_kafka import avro as avro
+from confluent_kafka import avro, TopicPartition
 
-record_schema_str = """
+
+# UserRecord represnts a user type which can be encoded and decoded as Avro.
+class UserRecord(dict):
```

Comment:
Suggest not using a dict since it is too forgiving (a dict matches any complex avro type).

Reply:
Right I guess I should stop fighting the fact that the python avro implementation doesn't support `specific` serialization/deserialization